sagan-site/src/main/java/sagan/projects/support/ProjectMetadataController.java

```diff
@@ -33,4 +40,87 @@ public Project projectMetadata(@PathVariable("projectId") String projectId) thro
         return service.getProject(projectId);
     }
 
+    @RequestMapping(value = "/{projectId}/**", method = GET)
```

Comment:
Tiny snit, but I kind of prefer it when it reads...

```
@RequestMapping(method = GET, value = "/{projectId}/**")
```

That way, it reads like a REST call, **GET /whatever**.